ui/app/AppLayouts/Onboarding/views/InsertDetailsView.qml

```diff
@@ -116,6 +116,7 @@ Item {
                     nameInput.input.edit.clear();
                 }
             }
+            validators: Constants.validators.displayName
```

Comment:
Great abstraction! It could be applied as well in `AppLayouts/Onbording/popups/GenKeyModal.qml`. `Input` should be replaced to `StatusInput` and just add the same validators. 
Created [task](https://github.com/status-im/status-desktop/issues/5299) to address that!